xml/System.Globalization/CultureInfo.xml

```diff
@@ -112,7 +112,7 @@
   
  Certain predefined culture names and identifiers are used by this and other classes in the <xref:System.Globalization?displayProperty=nameWithType> namespace. For detailed culture information for Windows systems, see the **Language tag** column in the [list of language/region names supported by Windows](https://docs.microsoft.com/openspecs/windows_protocols/ms-lcid/a9eac961-e77d-41a6-90a5-ce1a8b0cdb9c). Culture names follow the standard defined by [BCP 47](https://tools.ietf.org/html/bcp47).  
   
- Remember that the culture names and identifiers represent only a subset of cultures that can be found on a particular computer. Windows versions or service packs can change the available cultures. Applications add custom cultures using the <xref:System.Globalization.CultureAndRegionInfoBuilder> class. Users add their own custom cultures using the Microsoft Locale Builder tool. Microsoft Locale Builder is written in managed code using the `CultureAndRegionInfoBuilder` class.  
```

Comment:
>  [Microsoft Locale Builder](https://www.microsoft.com/en-us/download/details.aspx?id=41158) is written in managed code using the `CultureAndRegionInfoBuilder` class.

I am not sure if this is true or even add anything to the reader. We should remove this.